setup.py

```diff
@@ -24,33 +19,8 @@ def is_clang(bin):
     return not re.search(r'clang', output) is None
 
 
-def check_cython():
```

Comment:
Is it OK to delete this?

Reply:
Yes, because we now check build requirements with pyproject.toml